### DISCUSSION TITLE
How to set bias for secondary windows/panes for tall layout in session files?

### DISCUSSION THREAD

**Discussion by cybardev** [kovidgoyal/kitty#3237] [Q&A] [answered]
As far as I can see, the bias option in session files only affects the full_size window. The horizontal panes to the right are divided equally. I want to define a session file that looks like the attached screenshot.

I wish there was a way to set the vertical bias of secondary panes in a session file so I wouldn't have to manually resize them. Manually resizing takes too much time and effort.

![image](https://user-images.githubusercontent.com/50134239/104338022-48957180-5520-11eb-835c-a100cc1c9569.png)

Here,
pane1 = ~60% bias (horizontal width)
pane2 = ~63% bias (vertical height)
pane3 = remainder

**Comment by kovidgoyal** [comment]
That would require adding support for a --bias option to the launch command and adding support for the full aunch command syntax to the session files. At the moment your best bet is to use the remote control kitty @resize-window command in your session file or startup scripts.

**Reply by cybardev** [reply]
I will do that. Thank you. 😊

**Reply by cybardev** [reply]
That worked perfectly. Thanks again. Cheers. 😀

**Comment by cybardev** [comment, marked as answer]
`kitty @ resize-window` may have been the answer in 2021, but now there's a simpler way:
```bf
layout tall:bias=64
launch
launch
launch --bias=-14
```
(numbers to be adjusted as needed)

Either negative number support was added later, or due to my lack of wisdom at the time I didn't think of trying it out.
